lib/types/src/store_id.rs

```diff
@@ -11,6 +12,13 @@ use std::{
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
 pub struct StoreId(NonZeroUsize);
 
+impl StoreId {
+    /// Represents an unknown store ID
+    pub fn unknown() -> Self {
+        Self(usize::MAX.try_into().unwrap())
+    }
```

Comment:
This causes `Store::unknown() == Store::unknown()` which seems a bit strange for me

Reply:
let me take a look at that

Reply:
deleted